assets/javascripts/modules/checkout/validations.js

```diff
@@ -1,8 +1,18 @@
-define(['$', 'modules/checkout/form-elements', 'modules/checkout/regex'], function ($, form, regex) {
+define([
+    '$',
+    'modules/checkout/form-elements',
+    'modules/checkout/email-check',
+    'modules/checkout/regex'
+], function ($,
+             form,
+             emailCheck,
+             regex) {
     'use strict';
 
     var ERROR_CLASS = 'form-field--error',
-        EMAIL_ERROR_DEFAULT = 'Please enter a valid Email address';
+        EMAIL_ERROR_DEFAULT = 'Please enter a valid Email address',
+        EMAIL_ERROR_TAKEN = 'Your email is already in use! Please sign in or use another email address.',
+        EMAIL_ERROR_NETWORK = 'Your email could not be verified, please check your Internet connection and try again later.';
```

Comment:
Have the validation messages been written? If not can we get someone to provide the copy? I don't think the `EMAIL_ERROR_NETWORK` is one we want to display to the user.